tests/test_integration.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pytest
+
+
+@pytest.fixture
+def stubs(testdir):
+    tests = {
+        "inject": """def test_injection(snapshot):
```

Comment:
might be easier to read if we start it on a newline:
```python
"inject": (
    """
    def test_injection(snapshot):
        assert snapshot is not None
    """
)
```

and then trim space if necessary

Reply:
Done